src/pages/Services.js

```diff
@@ -19,9 +20,11 @@ class ServicesIndex extends Component {
 
   componentDidMount() {
     axios
-      .get(`${process.env.REACT_APP_CMS_ENDPOINT}/pages/?type=base.ServicePage`)
+      .post(`${process.env.REACT_APP_CMS_ENDPOINT}/graphql/`, {
+        query: allServicePagesQuery,
+      })
       .then(res => {
-        this.setState({ data: res.data })
+        this.setState({ data: res.data.data.allServicePages })
```

Comment:
is `data.data` code smell? 🧀

Reply:
ugh, it's terrible. `axios` uses `.data` as "the data that's returned in the response" and graphql's default thing is to return a `data` key. I could do more destructuring somewhere else but we're kinda stuck with `.data.data` somehow.